projects/mongoose/build.sh

```diff
@@ -13,5 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 cd $SRC/mongoose
 $CXX $CXXFLAGS $LIB_FUZZING_ENGINE -DMG_ENABLE_LINES -DMG_ENABLE_LOG=0 mongoose.c -I. test/fuzz.c -o $OUT/fuzz
+
+# Fuzzer using honggfuzz netdriver.
+if [[ "$FUZZING_ENGINE" == "honggfuzz" ]]
+then
+  $CC $LIB_FUZZING_ENGINE $CFLAGS -DMG_ENABLE_LINES=1 \
+    -I /src/honggfuzz/includes/ -D __NO_STRING_INLINES \
```

Comment:
Can we just include this -I flag as part of `HF_NETDRIVER_CFLAGS` (see above comment)?